src/app/core/service/gateway.service.ts

```diff
@@ -24,6 +25,12 @@ export class GatewayService {
       );
   }
 
+  getHistoriesList(limit: string = '5', page: string = '1', locale: string = 'uk') {
```

Comment:
```suggestion
  getHistoriesList(limit: string = '10', page: string = '1', locale: string = 'uk') {
```
It could include "load more" functionality, but for now load 10 items is enough.